index.html

```diff
@@ -35,5 +35,153 @@ <h1>Norbert Siko</h1>
+      <div class="proj-1">
+        <img src="assets/Proj1.png" alt="project 1" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj2.png" alt="project 2" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj3.png" alt="project 3" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj4.png" alt="project 4" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj5.png" alt="project 5" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+
+      <div class="proj-1">
+        <img src="assets/Proj6.png" alt="project 6" />
+        <div class="description">
+          <h1>Project name goes here</h1>
+          <ul>
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+          <button type="button">
+            See this project
+            <img src="assets/Union.png" alt="right arrow icon" />
+          </button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about">
+      <div class="about-description">
+        <h1>About</h1>
```

Comment:
- [ ] Just as I suggest above You should use an `h2 or h3` tag instead of an h1. You are meant to have only 1 `h1` on your page and that should be the main title. By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results. :+1: :+1: